archetype/config/env/production.js

```diff
@@ -1,37 +1,3 @@
-'use strict'
-
-const winston = require('winston')
-
 module.exports = {
 
-  trailpack: {
```

Comment:
For logger I personally use it as I run my projects into raspberry pi or stuff like that.
For the disabled option I hope we'll be able to make it work as it's useful in my opinion